File: go/largestandSecondLargest.go
package main

import "math"

func getLargestElement(arr []int) (int, float64) {
	largest := arr[0]
	secondLargest := math.Inf(-1)

	for i := 0; i <= len(arr)-1; i++ {
		if arr[i] > largest {
			secondLargest = float64(largest)
			largest = arr[i]
		}
	}

	return largest, secondLargest
}

File: python/largestandSecondLargest.py
def getLargestNmber(arr: list):
    largest = arr[0]
    
    if len(arr) == 0:
        return 0
    
    for num in range(len(arr)):
        if arr[num] > largest:
            largest = arr[num]
            
    return largest
            
numbers = [1, 2, 3, 99, 4, 101, 0, 5, 6, 9, 3]           
result = getLargestNmber(numbers)
print(result)


def getSecondLargest(arr: list):
    largest = arr[0]
    secondLargest = float('-inf')
    
    for num in range(len(arr)):
        if arr[num] > largest:
            secondLargest = largest
            largest = arr[num]
            
        elif arr[num] < largest and arr[num] > secondLargest:
            secondLargest = arr[num]
            
    return largest, secondLargest


marks = [11,0,3,100, 897, 888, 0,1,2,7,98]
result = print(getSecondLargest(marks))